internal/printer/ingress.go

```diff
@@ -54,6 +55,15 @@ func IngressListHandler(ctx context.Context, list *extv1beta1.IngressList, optio
 	return ot.ToComponent()
 }
 
+func getHostComponent(link string) component.Component {
+	_, err := url.Parse(link)
+	if strings.Contains(link, ".") && err == nil {
+		return component.NewLink("", link, "http://"+link)
```

Comment:
Can we look at the Ingress and determine if it is TLS/SSL?

Reply:
@mklanjsek ping

Reply:
Added TLS handling, also cases when multiple Urls are present.